Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -44,24 +45,13 @@ final class NetworkProtectionConnectionTester {
     /// The reason why this is necessary is that the tester may be stopped while the connection tests are already executing, in a bit
     /// of a race condition which could result in the tester returning results when it's already stopped.
     ///
-    private actor TimerRunCoordinator {
-        private(set) var isRunning = false
-
-        func start() {
-            isRunning = true
-        }
-
-        func stop() {
-            isRunning = false
-        }
-    }
```

Comment:
This coordinator was badly built (by me), and was giving us a fake sensation of thread safety.  Afuera!